x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_details/monitor_status/monitor_status_legend.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { css } from '@emotion/css';
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiText, useEuiTheme } from '@elastic/eui';
+import * as labels from './labels';
+import {
+  DANGER_VIZ_COLOR,
+  getErrorVizColor,
+  getSkippedVizColor,
+  SUCCESS_VIZ_COLOR,
+} from './monitor_status_data';
+
+export const MonitorStatusLegend = ({ brushable }: { brushable: boolean }) => {
+  const { euiTheme } = useEuiTheme();
+
+  const LegendItem = useMemo(() => {
+    return ({
+      color,
+      label,
+      iconType = 'dot',
+    }: {
+      color: string;
+      label: string;
+      iconType?: string;
+    }) => (
+      <EuiFlexItem
+        css={css`
+          display: flex;
+          flex-direction: row;
+          gap: 2px;
+        `}
+        grow={false}
+      >
+        <EuiIcon type={iconType} color={color} />
+        <EuiText size="xs">{label}</EuiText>
+      </EuiFlexItem>
```

Comment:
Maybe use https://elastic.github.io/eui/#/display/health component here

Reply:
The `EuiHealth` component doesn't accept _iconType_ parameter which we need to show the error type legend item (alert icon), so I'll leave it to the current implementation.